framework/core/src/Foundation/Console/InfoCommand.php

```diff
@@ -168,4 +186,27 @@ private function identifyDatabaseVersion(): string
     {
         return $this->db->getPdo()->getAttribute(PDO::ATTR_SERVER_VERSION);
     }
+
+    /**
+     * Some extensions/packages might be overriding the session.handler binding.
+     * So we have to check if the driver used is configured or hardcoded.
+     */
+    private function identifySessionDriver(): string
+    {
+        $defaultDriver = $this->session->getDefaultDriver();
+        $driver = Arr::get($this->config, 'session.driver', $defaultDriver);
+        $this->session->driver($driver);
```

Comment:
Not a big fan of effectful code in otherwise pure functions, can we add a comment explaining what this does, if we need it?

Reply:
good catch actually, should have adapted this when I made the other changes.